kbase-extension/static/kbase/js/widgets/appWidgets2/validation.js

```diff
@@ -610,9 +610,15 @@ define([
                     } else {
                         diagnosis = Constants.DIAGNOSIS.OPTIONAL_EMPTY;
                     }
-                } else if (options.values) {
+                } else if (options.values || options.options) {
+                    let targetSet;
+                    if (options.values) {
+                        targetSet = options.values;
+                    } else {
+                        targetSet = options.options.map((opt) => opt.value);
+                    }
```

Comment:
make `validateTextSet` work if the values you're supplying are from `dropdown_options`